app/controllers/org_admin/plans_controller.rb

```diff
@@ -54,6 +52,7 @@ def download_plans
 
     org = current_user.org
     file_name = org.name.gsub(/ /, "_")
+                        .gsub(/[\.;,]/, "")
```

Comment:
orgs with wierd chars in their names I take it?

Reply:
yep. E.g. 'University of California, Office of the President' and 'Univ. of So and So'